source/game.py

```diff
@@ -0,0 +1,26 @@
+from sliding_puzzle import Puzzle
+
+
+class Game:
+    """Game class"""
+
+    def __init__(self, image: str, difficulty: int = 1):
+        """Init function
+
+        Args:
+            image (str): image path
+            difficulty (int, optional): integer ranging from 1 to 5. Defaults to 1.
+        """
```

Comment:
Just aesthetics, but In the end all our doctstrings should use the same style
```suggestion
        :param image: image path
        :param difficulty: integer ranging from 1 to 5. Defaults to 1.
        """
```